frontend/schemaCMS/src/routes/page/editPage/editPage.component.js

```diff
@@ -148,6 +151,19 @@ export const EditPage = ({
     }
   };
 
+  const handleConfirmLeavePage = async () => {
+    try {
+      setPublishLoading(true);
+      history.push(customLocation);
```

Comment:
After we set a new location, the component is unmounted so I don't think we need to set any state after `history.push`

Reply:
fixed